crates/sui-protocol-constants/src/lib.rs

```diff
@@ -18,6 +18,9 @@ pub const MAX_MOVE_PACKAGE_SIZE: u64 = 100 * 1024; // 100 KB
 /// Maximum number of gas units that a single MoveCall transaction can use. Enforced by the Sui adapter.
 pub const MAX_TX_GAS: u64 = 1_000_000_000;
 
+/// Maximum time for bytecode verification in microseconds. Enforced by the adapter on the Move bytecode verifier.
+pub const MAX_VERIFIER_TIME_US: u64 = 20_000;
```

Comment:
I kept reading milliseconds and that is where I was massively confused - lol....
all good

Reply:
How is this number decided? It would be useful to add a metric first to measure how long bytecode verification currently takes, and look at p99/max value on devnet.

Reply:
> How is this number decided? It would be useful to add a metric first to measure how long bytecode verification currently takes, and look at p99/max value on devnet.

The number was decided by measuring the time for framework (~8.3ms) and other packages to verify. We then discussed internally and decided on 20ms being fair.

Reply:
Assume these experiments are run in a relatively idle environment? 
It's still not too late to add a metric, I think, to track the verification time as well as number of times we actually timeout.

Reply:
The `Duration` constructor functions are all `const fn` so you can use them here as well, to avoid the units-in-name pattern here as well.